app/src/main/java/io/appium/android/apis/app/IncomingMessage.java

```diff
@@ -121,12 +121,13 @@ void showAppNotification() {
         // The ticker text, this uses a formatted string so our message could be localized
         String tickerText = getString(R.string.imcoming_message_ticker_text, message);
 
-        // construct the Notification object.
-        Notification notif = new Notification(R.drawable.stat_sample, tickerText,
-                System.currentTimeMillis());
-
-        // Set the info for the views that show in the notification panel.
-        notif.setLatestEventInfo(this, from, message, contentIntent);
+        Notification notif = new Notification.Builder(this)
```

Comment:
We need to add channel like https://github.com/appium/android-apidemos/compare/master...KazuCocoa:km/update-demo-app?expand=1#diff-c8a8da14fd33ee6028b82302ad038b64R107

I've also updated as the branch yesterday and tested the demo app a bit
Created a PR => https://github.com/appium/android-apidemos/pull/34/files